documentation/src/main/asciidoc/user_guide/clustering.adoc

```diff
@@ -1086,6 +1092,15 @@ dcc.clustering().partitionHandling()
                     .whenSplit(PartitionHandling.ALLOW_READ_WRITES)
                     .mergePolicy(new CustomMergePolicy());
 ----
+==== Deploy custom merge policies to a Infinispan server instance
+If custom merge-policy is configured inside of the server configuration the classes need to be available whithin all servers of this cluster.
+For each policy an implementation of the EntryMergePolicy interface is needed.
```

Comment:
I think this is already covered in the [Merge Policies section](http://infinispan.org/docs/9.2.x/user_guide/user_guide.html#merge-policies).